apps/bond/common-util/functions/web3.js

```diff
@@ -1,17 +1,17 @@
+import { createPublicClient, getContract as getWagmiContract, http } from 'viem';
```

Comment:
```suggestion
import { createPublicClient, getContract as getViemContract, http } from 'viem';
```
nitpicking here..  but wagmi just provides the hooks, viem provides the contract instance